hazelcast/src/test/java/com/hazelcast/map/impl/operation/QueryPartitionOperationTest.java

```diff
@@ -0,0 +1,52 @@
+package com.hazelcast.map.impl.operation;
+
+import com.hazelcast.map.QueryResultSizeExceededException;
+import com.hazelcast.map.impl.QueryResult;
+import com.hazelcast.query.TruePredicate;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class QueryPartitionOperationTest extends QueryOperationTestSupport {
+
+    @Override
+    protected AbstractMapOperation createQueryOperation() {
+        return new QueryPartitionOperation(MAP_NAME, TruePredicate.INSTANCE);
+    }
+
+    @Test
+    public void testConstructor() {
+        new QueryPartitionOperation();
```

Comment:
This doesn't offer a lot of value. You not doing any validation on the fields. This can be relevant in some cases. So either validate the fields or remote the test method